tests/cypress/integration/actions_users/registration_involved/case_4_assign_taks_job_users.js

```diff
@@ -70,39 +86,93 @@ context('Multiple users. Assign task, job.', () => {
             );
             cy.logout(thirdUserName);
         });
-        it('First user login and create a task', () => {
+        it('First user login, create a task and logout', () => {
+            if (Cypress.browser.family !== 'chromium') {
+                cy.get('.cvat-modal-unsupported-platform-warning').within(() => {
+                    cy.contains('button', 'OK').click();
+                });
+            }
```

Comment:
Somewhere these lines before cy.login(), somewhere after. What it depends on?

Reply:
These actions moved in cy.logout().